### DISCUSSION TITLE
Plan for Power Ocean?

### DISCUSSION THREAD

**Discussion by birkej** [tolwi/hassio-ecoflow-cloud#158] [Ideas]
Hi,

I'm planning for a new PV System and would like to use the power ocean.
Do you know if this will be also supported in future?

THX

**Comment by Funki0816** [comment]
Hi, 

I bought a Powerocean System. Will it be supportet in the near future?

**Comment by Darkstar-74** [comment]
I would also be interressted and in a Support of the Power ocean System.

**Comment by LuxorN09** [comment]
Same here. 
There is a "clone" for Powerocean here (https://github.com/niltrip/powerocean) but it would be great to have it included in the "original" implementation.

**Comment by Bombarde16** [comment]
Me too...

**Comment by niltrip** [comment]
Me too... I think there is still no official api (no readable values from an mqtt Broker). I tried with this integration but i fail. I think for the moment it could be the same way like the Power Stream but i fail with the proto buff stuff.

**Comment by hellms** [comment]
+1 vote for powerocean
cheers

**Reply by michael15831** [reply]
I vote for PowerOcean too

**Comment by martinarlowski** [comment]
+1 vote for powerocen, please...

**Comment by Xygen** [comment]
On November 23rd there was an update to the [readme.md](https://github.com/tolwi/hassio-ecoflow-cloud/blob/main/README.md) mentioning PowerOcean via Public API.
So I tried version 1.4.0-beta3, but the device type for PowerOcean is missing from the drop down menu.
<img width="346" height="346" alt="image" src="https://github.com/user-attachments/assets/fea83fa0-0d34-41cc-958d-f4e10be6b038" />
Looking forward!

**Reply by florian4meier** [reply]
It is still missing isn't it? The Integration correctly identifies it as PowerOcean and it even identifies my PowerPulse, but I cannot integrate it due to the missing dropdown item

**Comment by Xygen** [comment]
Yes! PowerOcean support has been committed by @jumoog with PR #610 😍
Looking forward to the release! Awesome! Thank you!

**Reply by jumoog** [reply]
🕺

**Reply by jumoog** [reply]
Its dynamic so 1-2 string and 0-9 batteries

**Reply by Xygen** [reply]
Waiting for the release 🙏
